dotnet/src/dotnetcore/Providers/Messaging/GXAzureServiceBus/ServiceBusMessageBrokerProvider.cs

```diff
@@ -0,0 +1,130 @@
+			ReceiverOptions options = TransformGXUserTypeToReceiverOptions(receiverOptions);
+
+			GXProperties properties = new GXProperties();
+			properties.Add(PropertyConstants.MESSAGEBROKER_AZURESB_QUEUENAME, queueName);
+			properties.Add(PropertyConstants.MESSAGEBROKER_AZURESB_CONNECTIONSTRING, connectionString);
+			properties.Add(PropertyConstants.SESSION_ENABLED, sessionEnabled.ToString());
+			properties.Add(PropertyConstants.RECEIVE_MODE, options.ReceiveMode.ToString());
+			properties.Add(PropertyConstants.PREFETCH_COUNT, options.PrefetchCount.ToString());
+			properties.Add(PropertyConstants.RECEIVER_IDENTIFIER, options.Identifier);
+			properties.Add(PropertyConstants.RECEIVER_SESSIONID, options.SessionId);
+			properties.Add(PropertyConstants.SENDER_IDENTIFIER, senderIdentifier);
+
+			MessageQueue messageQueue = messageBrokerProvider.Connect(PropertyConstants.AZURESERVICEBUS, properties, out GXBaseCollection<SdtMessages_Message> errorMessagesConnect, out bool successConnect);
+			errorMessages = errorMessagesConnect;
+			success = successConnect;
+			return messageQueue;
+		}
+
+		public MessageQueue Connect(string topicName, string subcriptionName, string connectionString, bool sessionEnabled, GxUserType receiverOptions, string senderIdentifier, out GXBaseCollection<SdtMessages_Message> errorMessages, out bool success)
+		{
+			MessageBrokerProvider messageBrokerProvider = new MessageBrokerProvider();
+			GXProperties properties = new GXProperties();
+			ReceiverOptions options = TransformGXUserTypeToReceiverOptions(receiverOptions);
+
+			properties.Add(PropertyConstants.MESSAGEBROKER_AZURESB_QUEUENAME, topicName);
+			properties.Add(PropertyConstants.MESSAGEBROKER_AZURESB_SUBSCRIPTION_NAME, subcriptionName);
+			properties.Add(PropertyConstants.MESSAGEBROKER_AZURESB_CONNECTIONSTRING, connectionString);
+			properties.Add(PropertyConstants.SESSION_ENABLED, sessionEnabled.ToString());
+			properties.Add(PropertyConstants.RECEIVE_MODE, options.ReceiveMode.ToString());
+			properties.Add(PropertyConstants.PREFETCH_COUNT, options.PrefetchCount.ToString());
+			properties.Add(PropertyConstants.RECEIVER_IDENTIFIER, options.Identifier);
+			properties.Add(PropertyConstants.RECEIVER_SESSIONID, options.SessionId);
+			properties.Add(PropertyConstants.SENDER_IDENTIFIER, senderIdentifier);
+			
+			MessageQueue messageQueue = messageBrokerProvider.Connect(PropertyConstants.AZURESERVICEBUS, properties, out GXBaseCollection<SdtMessages_Message> errorMessagesConnect, out bool successConnect);
+			errorMessages = errorMessagesConnect;
+			success = successConnect;
+			return messageQueue;
+		}
+
+		#region Transformation methods
+		private ReceiverOptions TransformGXUserTypeToReceiverOptions(GxUserType options)
+		{
+			ReceiverOptions receiverOptions = new ReceiverOptions();
+			if (options != null)
+			{ 
+				receiverOptions.ReceiveMode = options.GetPropertyValue<short>("Receivemode");
```

Comment:
Receivemode is lowercase here. And camelcase on line 120: AddObjectProperty("ReceiveMode", ReceiveMode, false);
Maybe this is an error? IF not an error, ignore.

Reply:
No, it has no relation..